docs/toc.html

```diff
@@ -86,26 +86,26 @@
+                <li><a href="#georeplication">6.3 Geo-Replication (Cross-Cluster Data Mirroring)</a></li>
                     <ul>
-			<li><a href="#georeplication-overview">Geo-Replication Overview</a></li>
-			<li><a href="#georeplication-flows">What Are Replication Flows</a></li>
-			<li><a href="#georeplication-mirrormaker">Configuring Geo-Replication</a></li>
-			<li><a href="#georeplication-starting">Starting Geo-Replication</a></li>
-			<li><a href="#georeplication-stopping">Stopping Geo-Replication</a></li>
-			<li><a href="#georeplication-apply-config-changes">Applying Configuration Changes</a></li>
-			<li><a href="#georeplication-monitoring">Monitoring Geo-Replication</a></li>
+                        <li><a href="#georeplication-overview">Geo-Replication Overview</a></li>
+                        <li><a href="#georeplication-flows">What Are Replication Flows</a></li>
+                        <li><a href="#georeplication-mirrormaker">Configuring Geo-Replication</a></li>
+                        <li><a href="#georeplication-starting">Starting Geo-Replication</a></li>
+                        <li><a href="#georeplication-stopping">Stopping Geo-Replication</a></li>
+                        <li><a href="#georeplication-apply-config-changes">Applying Configuration Changes</a></li>
+                        <li><a href="#georeplication-monitoring">Monitoring Geo-Replication</a></li>
                     </ul>
-		<li><a href="#multitenancy">6.4 Multi-Tenancy</a></li>
+		        <li><a href="#multitenancy">6.4 Multi-Tenancy</a></li>
                     <ul>
-			<li><a href="#multitenancy-overview">Multi-Tenancy Overview</a></li>
-			<li><a href="#multitenancy-topic-naming">Creating User Spaces (Namespaces)</a></li>
-			<li><a href="#multitenancy-topic-configs">Configuring Topics</a></li>
-			<li><a href="#multitenancy-security">Securing Clusters and Topics</a></li>
-			<li><a href="#multitenancy-isolation">Isolating Tenants</a></li>
-			<li><a href="#multitenancy-monitoring">Monitoring and Metering</a></li>
-			<li><a href="#multitenancy-georeplication">Multi-Tenancy and Geo-Replication</a></li>
-			<li><a href="#multitenancy-more">Further considerations</a></li>
+                        <li><a href="#multitenancy-overview">Multi-Tenancy Overview</a></li>
+                        <li><a href="#multitenancy-topic-naming">Creating User Spaces (Namespaces)</a></li>
+                        <li><a href="#multitenancy-topic-configs">Configuring Topics</a></li>
+                        <li><a href="#multitenancy-security">Securing Clusters and Topics</a></li>
+                        <li><a href="#multitenancy-isolation">Isolating Tenants</a></li>
+                        <li><a href="#multitenancy-monitoring">Monitoring and Metering</a></li>
+                        <li><a href="#multitenancy-georeplication">Multi-Tenancy and Geo-Replication</a></li>
+                        <li><a href="#multitenancy-more">Further considerations</a></li>
```

Comment:
This change is only cosmetic on the source part, as now the subsections 6.2, 6.3 and 6.4 are aligned at the same column.